### PR TITLE
vagrant: add windows 2016 support and action

### PR DESCRIPTION
**Pull request by gdams**
##### Checklist

- [x] commit message has one of the [standard prefixes](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/FAQ.md#commit-messages)
- [ ] [FAQ.md](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

**Comment by Willsparker** [comment]
Judging from the check, it looks like this is required for 2016 as well:
https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/07ec26d21f70b810dbc0589b3d3b3ee01de5adf7/ansible/Vagrantfile.Win2012#L12

**Comment by Willsparker** [comment]
Ref: The `become` issue: https://groups.google.com/g/ansible-project/c/r52FeZLjttI?pli=1

It's weird that you'd need to specify `runAs`, considering we don't do that for 2012r2. 
One other thing I'd mention - the playbook appears to be running on every single port that the VM has opened, however, it should just need `55986` (the last port number given when running `vagrant port`).

**Comment by karianna** [comment]
```
TASK [Gathering Facts] *********************************************************
fatal: [127.0.0.1]: FAILED! => {"msg": "The powershell shell family is incompatible with the sudo become plugin"}

PLAY RECAP *********************************************************************
127.0.0.1                  : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   

Error: Process completed with exit code 2.
```

File: .github/workflows/build_vagrant_win.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        if ! grep -q "credssp" playbooks/AdoptOpenJDK_Windows_Playbook/group_vars/all/adoptopenjdk_variables.yml;
+        then
+          # Add the "ansible_winrm_transport" to adoptopenjdk_variables.yml
+          echo -e "\nansible_winrm_transport: credssp" >> playbooks/AdoptOpenJDK_Windows_Playbook/group_vars/all/adoptopenjdk_variables.yml
+        fi
+
+    - name: Run Ansible Playbook
+      run: |
+        cd ansible
+        ansible-playbook -i playbooks/AdoptOpenJDK_Windows_Playbook/hosts.win -u vagrant -b --skip-tags adoptopenjdk playbooks/AdoptOpenJDK_Windows_Playbook/main.yml
```
Comment (Willsparker) [inline .github/workflows/build_vagrant_win.yml:49]:
```suggestion
        ansible-playbook -i playbooks/AdoptOpenJDK_Windows_Playbook/hosts.win -u administrator --skip-tags adoptopenjdk playbooks/AdoptOpenJDK_Windows_Playbook/main.yml
```
Had a quick check, the Administrator account has the same password as the vagrant account (i.e. `vagrant`), and stops the need for the `-b` tag, that's causing an issue :-)

**Review by Willsparker — CHANGES_REQUESTED** [review]

**Review by Willsparker — COMMENTED** [review]
LGTM, in terms of code - The GHCheck has still failed, with an error I see fairly often in VPC. I'm somewhat concerned to have this as a GHCheck due to this instability of it

**Comment by gdams** [comment]
@Willsparker removed the GH action

**Review by Willsparker — APPROVED** [review]
LGTM - I'm not entirely against adding it in the future - I'm looking at what could make the win2012r2 VF buggy in VPC now with #2056 , so we may be able to apply those changes / Fixes to 2016 too

**Pull request merged**